### PR TITLE
Update quick-overview.md

### PR DESCRIPTION
**Pull request by ali-rostami**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [x] Update the documentation

### CODE REVIEW

**Comment by ali-rostami** [comment]
fix a typo.

File: docs/overview/coords.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ For example to align a label above a button and center the label is horizontally
 lv_obj_align_to(label, btn, LV_ALIGN_OUT_TOP_MID, 0, -10);
 ```
 
-Not that - unlike with `lv_obj_align()` - `lv_obj_align_to()` can not realign the object if its coordinates or the reference object's coordinates changes. 
+Note that - unlike with `lv_obj_align()` - `lv_obj_align_to()` can not realign the object if its coordinates or the reference object's coordinates changes.
```
Comment (ali-rostami) [inline docs/overview/coords.md:151]: In previous versions this was possible, right?

Comment (kisvegabor) [reply docs/overview/coords.md:151]: In v7 there was `obj_realign` that could realign on obj's size change. But didn't work on parent's size change.

**Comment by kisvegabor** [comment]
Thanks!

File: docs/overview/coords.md (hunk 1 above)
Comment (ali-rostami) [reply docs/overview/coords.md:151]: > In v7 there was `obj_realign` that could realign on obj's size change. But didn't work on parent's size change.

thanks, oops, so then what is the `obj_realign` replacement?

Comment (kisvegabor) [reply docs/overview/coords.md:151]: Objects aligned on their parent with `lv_obj_align` are realigned automatically but there is no replacement when the object is aligned to another object using `lv_obj_align_to`. In such a case probably it's better to use a [layout](https://docs.lvgl.io/8.0/layouts/index.html).

**Pull request merged**